### ISSUE TITLE
"Problem saving workflow" when tags disabled

### ISSUE THREAD

**Issue by rizalfauzie** [n8n-io/n8n#3781]
**Describe the bug**
Got error message when saving workflow `Problem saving workflow` when workflow tag is disabled. But the workflow still saved on database after i reload and load from workflow menu, everything is okay.

**To Reproduce**
1. Clean install with latest version
2. Run with `export N8N_WORKFLOW_TAGS_DISABLED=true && n8n start`
3. Create new workflow
4. make some changes
5. Click Save button, on first time it's okay
6. Make some changes again
7. Then click Save again
8. The error message appear

**Expected behavior**
The success message after workflow is saved to database.

**Environment (please complete the following information):**
 - OS: Mac OS 12.1 Intel
 - n8n Version 0.187.2
 - Node.js Version 18.6.0
 - Database system SQLite
 - Operation mode own

**Comment by rizalfauzie** [comment]
Additional output from debug console log:
```
TypeError: Cannot read properties of undefined (reading 'length')
    at /Users/*****/.nvm/versions/node/v18.6.0/lib/node_modules/n8n/dist/src/Server.js:577:38
    at async /Users/*****/.nvm/versions/node/v18.6.0/lib/node_modules/n8n/dist/src/ResponseHelper.js:90:26
```

**Comment by brianinoa** [comment]
Thanks @rizalfauzie I'm taking a look at this

**Comment by janober** [comment]
The fix just got merged with #3792 and will be released with the next version.

**Comment by janober** [comment]
Fix got released with `n8n@0.189.0`
